### PR TITLE
Updated "Password Reset Protection Type" section

### PR DESCRIPTION
**Pull request by agorbivskyi**
## Purpose of this pull request

This pull request (PR) is intended to fix incorrect information in "Password Reset Protection Type"

## Affected documentation pages

https://docs.magento.com/user-guide/customers/password-options.html

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

File: src/customers/password-options.md [hunk 1]
```diff
@@ -15,12 +15,14 @@ The customer password options determine the level of security that is used for p
...
-   | By IP | The password can be reset online without additional confirmation. |
-   | By Email | The password can be reset only by responding to an email notification that is sent to the email address associated with the Customer account. |
-   | None | The password can be reset only by the store administrator. |
+1. Set the **Password Reset Protection Type** to the method you want to use for checking password reset requests:
+
+   | By IP and Email | Check for previous attempts to reset password for specific email or from specific IP. |
+   | By IP | Check for previous attempts to reset password from specific IP. |
+   | By Email | Check for previous attempts to reset password for specific email. |
+   | None | Protection disabled (no limits for resetting password). |
+
```
Comment (leslietilling) [inline src/customers/password-options.md:24]: This will fix your trailing space

```suggestion

```

File: src/customers/password-options.md [hunk 2]
```diff
@@ -21,7 +21,7 @@ The customer password options determine the level of security that is used for p
    | By IP | Check for previous attempts to reset password from specific IP. |
    | By Email | Check for previous attempts to reset password for specific email. |
    | None | Protection disabled (no limits for resetting password). |
-   Based on this configuration **Max Number of Password Reset Requests** and **Min Time Between Password Reset Requests** will be calculated.
```
Comment (leslietilling) [inline src/customers/password-options.md]: Must have a blank line (no spaces) after the table

File: src/customers/password-options.md [hunk 3]
```diff
@@ -15,12 +15,14 @@ The customer password options determine the level of security that is used for p
...
-   | By IP | The password can be reset online without additional confirmation. |
-   | By Email | The password can be reset only by responding to an email notification that is sent to the email address associated with the Customer account. |
-   | None | The password can be reset only by the store administrator. |
+1. Set the **Password Reset Protection Type** to the method you want to use for checking password reset requests:
+
+   | By IP and Email | Check for previous attempts to reset password for specific email or from specific IP. |
+   | By IP | Check for previous attempts to reset password from specific IP. |
+   | By Email | Check for previous attempts to reset password for specific email. |
+   | None | Protection disabled (no limits for resetting password). |
+
```
Comment (leslietilling) [inline src/customers/password-options.md:24]:
```suggestion

```

**Comment by leslietilling** [comment]
running tests

**Comment by leslietilling** [comment]
@agorbivskyi  Is this update specific to 2.4, or should we backport this to 2.3 also?

**Comment by agorbivskyi** [comment]
@leslietilling 2.3 and 2.4

**Pull request merged**